test/testcafe/spec/TerminalView_spec.js

```diff
@@ -199,40 +199,31 @@ test.requestHooks(endUserRemediationOneOptionMock)('should render end user remed
   const terminalViewPage = await setup(t);
   await checkA11y(t);
 
-  await t.expect(terminalViewPage.form.getErrorBoxHtml()).eql([
-    '<span data-se="icon" class="icon error-16"></span>',
```

Comment:
Thanks for updating this test, it's much better already now that we aren't asserting on exact HTML.